.vscode/launch.test.json

```diff
@@ -0,0 +1,28 @@
+{
+    "run": {
+        "default": "",
+        "items": [
+            {
+                "name": "xml-languageserver",
+                "projectName": "xml-languageserver",
+                "workingDirectory": "/home/nkomonen/Documents/angelo-xmlLS/xml-languageserver",
+                "args": [],
+                "vmargs": [],
+                "preLaunchTask": ""
+            }
+        ]
+    },
+    "debug": {
+        "default": "",
+        "items": [
+            {
+                "name": "xml-languageserver",
+                "projectName": "xml-languageserver",
+                "workingDirectory": "/home/nkomonen/Documents/angelo-xmlLS/xml-languageserver",
```

Comment:
"workingDirectory": ""

Reply:
I cleaned everything up and this file is not here anymore. Everything works fine though.